### DISCUSSION TITLE
Expand cross_sells.product with permissions

### DISCUSSION THREAD

**Discussion by soyzamudio** [swellstores/community#298] [Help] [answered]
I've been trying to expand `cross_sells.product` with the frontend api, I've tried opening content permissions but I've been unsuccessful at it, what would be the body in the API request in console to opening it up?

Also, how would I remove some permissions I have added (while trying this I have added several permissions that I would like to remove)

Thanks!

**Comment by soyzamudio** [comment]
I solved it now by fetching each product with the `productId`, but i would like to not make any unnecessary calls if I can get the product in the `cross_sells` property 🤔

**Reply by ericingram** [reply]
@logeshswell let's add `cross_sells.product` to the default frontend API permissions

**Reply by logeshswell** [reply, marked as answer]
@soyzamudio Following [this documentation](https://developers.swell.is/guides/opening-content-permissions#adjusting-the-permissions), you can provide the below request body to open-up permissions for cross-sells:
```json
{
  "permissions": {
    "products": {
      "fields": [
        "cross_sells.product",
        "cross_sells.product_id",
      ]
    }
  }
}
```

As per Eric advise, I will also include these permissions by default.
